LocalMultiplayerAgent/Program.cs

```diff
@@ -37,9 +36,14 @@ public static async Task Main(string[] args)
             // lcow stands for Linux Containers On Windows => https://docs.microsoft.com/en-us/virtualization/windowscontainers/deploy-containers/linux-containers
             Globals.GameServerEnvironment = args.Contains("-lcow") && RuntimeInformation.IsOSPlatform(OSPlatform.Windows)
                 ? GameServerEnvironment.Linux : GameServerEnvironment.Windows; // LocalMultiplayerAgent is running only on Windows for the time being
+            Globals.CreateDeployment = args.Contains("-deploy");
```

Comment:
as mentioned before, I'd strongly recommend using the "Build" keyword instead of "Deploy"

Reply:
FWIW, our APIs are named "CreateBuildXXX"